src/bin/vip-config-software-update.js

```diff
@@ -22,23 +22,38 @@ const UPDATE_SOFTWARE_PROGRESS_STEPS = [
 	{ id: 'process', name: 'Process software update' },
 ];
 
+const usage = 'vip config software update <wordpress|php|nodejs|muplugins> <version>';
+const exampleUsage = 'vip @example-app.develop config software update';
+const exampleUsageNode = 'vip @example-node-app.develop config software update';
+
+const examples = [
+	{
+		usage: `${ exampleUsage } wordpress 6.4`,
+		description: 'Update WordPress application environment to 6.4.x.',
+	},
+	{
+		usage: `${ exampleUsage } wordpress managed_latest`,
+		description:
+			'Update WordPress application environment to the latest major version of WordPress and automatically update when VIP makes the next version available.',
```

Comment:
Suggested:
`Update a WordPress environment to the latest major version of WordPress, and automatically update WordPress to the next major version on a continual basis.`